api/api-plugin-infos-v6/src/main/java/com/thoughtworks/go/apiv6/plugininfos/representers/extensions/ConfigRepoExtensionRepresenter.java

```diff
@@ -28,7 +28,10 @@ public void toJSON(OutputWriter extensionWriter, PluginInfo extension) {
 
         extensionWriter.addChild("capabilities", capabilitiesWriter ->
                 capabilitiesWriter.add("supports_pipeline_export", configRepoPluginInfo.getCapabilities().isSupportsPipelineExport())
-                        .add("supports_parse_content", configRepoPluginInfo.getCapabilities().isSupportsParseContent()));
+                        .add("supports_parse_content", configRepoPluginInfo.getCapabilities().isSupportsParseContent())
+                        .add("supports_list_config_files", configRepoPluginInfo.getCapabilities().isSupportsListConfigFiles())
+                        .add("supports_user_defined_properties", configRepoPluginInfo.getCapabilities().isSupportsUserDefinedProperties())
```

Comment:
@marques-work - here we're modifying the existing `Plugin Infos API v6`. 
Should we consider adding `Plugin Infos API v7`?

Reply:
Yes, you're right. I probably should have moved this to a new `v7`. Curious though, does anyone actually use this API? It feels very, very specialized/niche. I suppose that's a larger discussion though.